changelog/0.8.2/pull-1549

```diff
@@ -0,0 +1,9 @@
+Enchancement: Speed up querying across indices and speed up scanning existing files
```

Comment:
Typo: "Enhancement". That'll also fail on Travis when [calens](https://github.com/restic/calens) is run to generate the changelog file:
```
$ calens
file changelog/0.8.2/pull-1549: entry type "Enchancement" is invalid, valid types: map[Enhancement:4 Security:1 Bugfix:2 Change:3]
```